pkg/cmd/run.go

```diff
@@ -293,7 +317,7 @@ func fetchSecrets(localConfig models.ScopedOptions, enableCache bool, enableFall
 		utils.HandleError(err, "Unable to parse API response")
 	}
 
-	writeFallbackFile := enableFallback && !fallbackReadonly
+	writeFallbackFile := enableFallback && !fallbackReadonly && nameTransformer == nil
```

Comment:
We don't support fallback files with the name transformer?

Reply:
Mainly because it's technically challenging and we want to gauge interest on this feature before we fully build it out.

Specifically, if you run `doppler run -- env`, it'll save the resulting secrets to the fallback file without any name transformation. If you then run `doppler run --name-transform camel --fallback-only -- env`, it will provide you with the un-transformed secrets without an error. In order to fail properly in that case, we'd need to provide some info in the fallback file (or the path itself) which include the name transformer which was initially used. I initially planned to add it to the metadata file but supplying `--fallback-only` will bypass the cache check today.